lib/ansible/modules/cloud/amazon/ec2_asg.py

```diff
@@ -1622,6 +1639,12 @@ def asg_exists(connection):
     return bool(len(as_group))
 
 
+def fail_if_max_instance_lifetime_not_supported():
```

Comment:
You've not passed module in here.

Reply:
`module` is defined as a global already. I was thinking of converting this to an AnsibleAWSModule in a separate PR.